src/govuk/common/index.mjs

```diff
@@ -0,0 +1,149 @@
+/**
+ * TODO: Ideally this would be a NodeList.prototype.forEach polyfill
+ * This seems to fail in IE8, requires more investigation.
+ * See: https://github.com/imagitama/nodelist-foreach-polyfill
+ *
+ * @param {NodeListOf<Element>} nodes - NodeList from querySelectorAll()
+ * @param {nodeListIterator} callback - Callback function to run for each node
+ * @returns {undefined}
+ */
+export function nodeListForEach (nodes, callback) {
```

Comment:
Cheers for reorganising this 😊 

Thoughts on making the `index.mjs` file a namespace export? Typically we'd want `index.mjs` to export everything from the directory

Currently two imports are needed:

```mjs
import { closestAttributeValue } from './common/closest-attribute-value.mjs'
import { nodeListForEach } from './common/index.mjs'
```

Which we could turn into one?

```mjs
import { closestAttributeValue, nodeListForEach } from './common/index.mjs'
```

Reply:
Cheers for chatting through this

Yep, we'd be bringing back "side effects" by importing all those polyfills again

Feels like we _could_ import a "ready to run" polyfill but then we're back down the route of doing what [`core-js`](https://github.com/zloirock/core-js) will do for us:

```mjs
import { detect, polyfill } '../../vendor/polyfills/Element/prototype/closest.mjs'

if (!detect()) {
  polyfill()
}
```

I'm happy to leave this file as it is